node-api/src/error.rs

```diff
@@ -103,6 +103,7 @@ impl RuntimeError {
                 Ok(Self::Other("transactional error".into()))
             }
             DispatchError::Other(msg) => Ok(Self::Other(msg.to_string())),
+            _ => todo!(),
```

Comment:
This is only necessary because the substrate version has been updated (due to the new dependency). I'd rather have this update in a separate PR if possible. You can downgrade the substrate version easily with the command
`cargo update -p sp-std --precise f1c60e529a6c1d158a0a37fe0a8daf887b2b9f74`.

Reply:
Resolved